lib/parsers/tiff_parser.rb

```diff
@@ -28,20 +27,21 @@ def call(io)
     exif_data = exif_from_tiff_io(io)
     return unless exif_data
 
+    # arw files are now handled in separate arw_parser
```

Comment:
Also this comment seems a bit redundant, in my opinion.

Reply:
👍 I'll remove it